smb_slam/launch/localization.launch

```diff
@@ -5,24 +5,25 @@
   <arg name="use_sim_time"          default="false" />
   <arg name="cloud_topic"           default="/rslidar/points"  doc="input topic for the lidar pointcloud"/>
   <arg name="parameter_filename"    default="param_rs16_localization.lua" doc="param file name"/>
```

Comment:
please extend the doc string to point out that the pcd file is specified in the lua file.